client/src/main/java/io/obswebsocket/community/client/message/request/inputs/GetInputAudioMonitorTypeRequest.java

```diff
@@ -6,13 +6,13 @@
 
 @Getter
 @ToString(callSuper = true)
-public class GetInputTracksRequest extends InputRequest {
+public class GetInputAudioMonitorTypeRequest extends InputRequest {
```

Comment:
The `GetInputTracksRequest` still exists, please do not delete it

Reply:
Hmm, guess i missed that one